siddhi-extensions/geo/src/main/java/org/wso2/siddhi/gpl/extensions/geo/stream/GeoCrossesStreamProcessor.java

```diff
@@ -52,7 +52,7 @@
      * @return the additional output attributes introduced by the function
      */
     @Override
-    protected List<Attribute> init(AbstractDefinition inputDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext, boolean outputExpectsExpiredEvents) {
+    protected List<Attribute> init(AbstractDefinition inputDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext) {
         geoOperation = new WithinOperation();
         geoOperation.init(attributeExpressionExecutors, 1, attributeExpressionLength);
         ArrayList<Attribute> attributeList = new ArrayList<Attribute>(1);
```

Comment:
@ashensw It's preferable to code against the interface and not the implementation. Therefore, it's better to use `List<Attribute> attributeList` instead of `ArrayList<Attribute> attributeList`.